deploy/kubernetes/helm/che/values.yaml

```diff
@@ -34,6 +34,7 @@ global:
   ingressDomain: 192.168.99.100.nip.io
   # See --annotations-prefix flag (https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/cli-arguments.md)
   ingressAnnotationsPrefix: "nginx."
+  defaultIngressClass: "nginx"
```

Comment:
Please consider renaming it to `ingressClass` to be consistent with `ingressAnnotationsPrefix`.

Reply:
`defaultIngressClass` is from Operator implementation https://github.com/eclipse/che-operator/blob/fbeb80885d76f1dc1a662967f7a7277de7f9738c/pkg/deploy/defaults.go#L32

Reply:
Note that you referenced name variable in go code.
and when a user wants to specify ingress class, he defines `cr.Spec.K8SOnly.IngressClass` See https://github.com/eclipse/che-operator/blob/8837474a0fe4082679af4b2393ca55992cc5f75c/pkg/deploy/ingress.go#L29

In helm case, a user would define `defaultIngressClass`.
I still think that it makes sense to rename it to `IngressClass`

Reply:
yes, make sense. fixed.